app/dcs/api/s3/s3-compatibility/page.md

```diff
@@ -120,6 +120,19 @@ Full compatibility means that we support all features of a specific action
 except for features that rely on other actions that we haven't fully
 implemented.
 
+### GetBucketLocation
+
+This is currently supported in Gateway-MT only.
+
+#### LocationConstraint
+
+Specifies the placement region where the bucket data is stored:
+
+|     Value     |                                                        Description                                                        |
+|:-------------:|:-------------------------------------------------------------------------------------------------------------------------:|
+| `global`      | Stored on the entire globe connected to the public network                                                                |
```

Comment:
```suggestion
| `global`      | Stored on nodes connected to the global public network                                                                |
```

Reply:
It read a little bit weird. I think this is better, but I'm not SUPER opinionated here.